admin/fleetcommander/fcfreeipa.py

```diff
@@ -110,8 +108,8 @@ def _create_profile_rules(self, profile):
         api.Command.deskprofilerule_add(name, **parms)
 
         # Save rules for users
-        users = list(map(six.text_type, profile["users"]))
-        groups = list(map(six.text_type, profile["groups"]))
+        users = list(map(str, profile["users"]))
+        groups = list(map(str, profile["groups"]))
```

Comment:
list compehension is more readable and may be faster here:
```diff
- users = list(map(str, profile["users"]))
￼- groups = list(map(str, profile["groups"]))
+ users = [str(x) for x in profile["users"]]
+ groups = [str(x) for x in profile["groups"]]
```

Reply:
Resolved